.github/workflows/plugins-ci-postgres.yml

```
@@ -30,38 +35,64 @@ jobs:
       - name: Dependency review
         uses: actions/dependency-review-action@v1
 
+  license-check:
+    if: >
+      !failure() &&
+      !cancelled() &&
+      inputs.license-check == true
+    name: Check Licenses
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+    steps:
+      - uses: actions/checkout@v3.0.2
+        with:
+          persist-credentials: false
+
+      - name: Setup node 16
+        uses: actions/setup-node@v3.1.1
+        with:
+          node-version: 16
```

Probably better to use LTS so we don't have to keep updating the node-version for these actions. :)

```suggestion
      - name: Setup Node
        uses: actions/setup-node@v3
        with:
          node-version: lts/*
```